samples/sensor/die_temp_polling/boards/esp32s2_saola.overlay

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
```

Comment:
Any reason to remove current Copyright?

Reply:
In the past commit, it was pointed out that the copyright should be change, so I changed it this commit as well.
https://github.com/zephyrproject-rtos/zephyr/pull/55426#discussion_r1125418191

Reply:
IMO That was a different situation since you created a new file, unlike the current case.